src/test/java/dk/kb/present/transform/XSLTPreservicaSchemaOrgTransformerTest.java

```diff
@@ -31,6 +31,7 @@ public class XSLTPreservicaSchemaOrgTransformerTest {
     public static final String RECORD_9d9785a8 = "internal_test_files/tvMetadata/9d9785a8-71f4-4b34-9a0e-1c99c13b001b.xml";
     public static final String RECORD_c6fde2f4 = "internal_test_files/tvMetadata/c6fde2f4-036a-4e04-b83a-39a92021460b.xml";
     public static final String RECORD_68b233c3 = "internal_test_files/tvMetadata/68b233c3-f234-4546-914e-dc912f6001ae.xml";
+    public static final String RECORD_df3dc9cf = "internal_test_files/tvMetadata/df3dc9cf-43f6-4a8a-8909-de8b0fb7bd00.xml";
```

Comment:
Seems like you forgot to add this to aegis?

Reply:
Oops, added!